tests/test_filtsmooth/test_statespace/test_integrator.py

```diff
@@ -0,0 +1,193 @@
+import unittest
+
```

Comment:
the tests in this module are the same tests as previously in `ODEPrior`, with very few additions.